changelog/16826.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+website/docs: SSH CA Certificate notes clarifying how to preserve comment attributes in keys.
```

Comment:
Docs changes shouldn't have a changelog entry.

Reply:
Thanks - was just trying to get the check marks on the CI. Should I remove changelog file @cipherboy ?

Reply:
There's a magic tag "pr/no-changelog" I've just added for you - that will skip the test (so you won't have an ugly red x on the ci)

Reply:
Yes, please remove :-)